to-do-list/src/API/PostService.tsx

```diff
@@ -0,0 +1,39 @@
+import axios from 'axios'
```

Comment:
the .tsx uses for the components that returns JSX.Element. All the other files like interfaces, services should be with .ts
Also, you don't need to create a class if you only need to get functions. Just create functions with export statement, you'll be able to use it anywhere in the project
Also, it's better to specify names. You have PostService with create method and, just reading the names, I cannot explain that they should do. It would be better to create TaskService and use it like TaskService.create or rename also methods to createTask, updateTask and use them without the service name